js/src/data/constants.js

```diff
@@ -16,3 +16,9 @@ export const EMPTY_ASSET_ENTITY_GROUP = {
 	[ ASSET_GROUP_KEY.FINAL_URL ]: '',
 	[ ASSET_GROUP_KEY.DISPLAY_URL_PATH ]: [],
 };
+
+export const ERROR_SLOTS = {
+	GOOGLE_MC_CONNECTION: 'google_mc_connection',
```

Comment:
@ankitrox Let's add a prefix to the error slots in case we want to extend the error handling for other pages/sections in the future. Let's prefix the error slots with `setup-mc` which happens within onboarding. We can try to see how we can use the `CONTEXT_EXTENSION_ONBOARDING` [constant](https://github.com/woocommerce/google-listings-and-ads/blob/develop/js/src/pages/onboarding/setup-stepper/saved-setup-stepper.js#L29). Basically we should be able to uniquely identify the slots.